CMakeLists.txt

```diff
@@ -485,30 +485,32 @@ if(MSVC)
   add_msvc_runtime_flag(onnxifi_loader)
 endif()
 
-# ---[ ONNXIFI wrapper
-add_library(onnxifi_wrapper MODULE onnx/onnxifi_wrapper.c)
-target_include_directories(onnxifi_wrapper PRIVATE
-  $<BUILD_INTERFACE:${ONNX_ROOT}>
-  $<INSTALL_INTERFACE:include>)
-set_target_properties(onnxifi_wrapper PROPERTIES
-  C_STANDARD 99
-  C_EXTENSIONS NO
-  OUTPUT_NAME "onnxifi"
-  POSITION_INDEPENDENT_CODE YES)
-target_link_libraries(onnxifi_wrapper PRIVATE onnxifi_loader onnxifi)
-if(DEFINED ONNXIFI_SEARCH_DIR)
-  target_compile_definitions(onnxifi_wrapper PRIVATE "ONNXIFI_SEARCH_DIR=\"${ONNXIFI_SEARCH_DIR}\"")
-endif()
-if(WIN32)
-  if(MSVC)
-    target_compile_definitions(onnxifi_wrapper PRIVATE "ONNXIFI_PUBLIC=__declspec(dllexport)")
-  else()
-    target_compile_definitions(onnxifi_wrapper PRIVATE "ONNXIFI_PUBLIC=__attribute__((__dllexport__))")
+if (NOT (ANDROID OR IOS))
```

Comment:
IMO, `if (NOT ANDROID AND NOT IOS)` is clearer, but I don't have a strong opinion